### PR TITLE
Verify our tables have correct formatting - for consistency

### PR DESCRIPTION
**Pull request by duglin**
For those of us who are more AR than others :-)

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

**Comment by shalako** [comment]
LGTM

File: spec.md [hunk 1]
```diff
@@ -451,7 +451,7 @@ $ curl http://username:password@broker-url/v2/service_instances/:instance_id?acc
 ### Response
 
 | Status Code | Description |
-|---|---|
+| --- | --- |
```
Comment (pmorie) [inline spec.md]: I have never seen spaces between the dashes and pipes in a markdown header before in my life.... citation?

#buildthebikeshed

Comment (duglin) [inline spec.md]: No idea what the correct format is but this was the only one that didn't have them so I decided this one was the odd man out.  :-)
There's this doc:  https://help.github.com/articles/organizing-information-with-tables/    which shows it with spaces.

**Comment by angarg12** [comment]
Agree with @pmorie. If that is going to be part of this PR I'll wait to LG it.

**Comment by duglin** [comment]
that is part of the PR already - unless I'm not following

**Comment by pmorie** [comment]
Wow, I did somehow miss the checker.

**Comment by angarg12** [comment]
LGTM

**Comment by duglin** [comment]
could use another round of reviews.

**Comment by avade** [comment]
lgtm

**Comment by shalako** [comment]
LGTM

**Comment by duglin** [comment]
@angarg12 your re-l.g.t.m is needed

**Comment by zrob** [comment]
LGTM

**Pull request merged**